packages/website/docs/Button.md

```diff
@@ -1,39 +1,43 @@
-| Combo     | These buttons have an attached drop menu where a choice between multiple actions has to be made (2 or more). They are useful when one single action allows users to create, delete or modify many different elements, which can be selected in the dropdown list that appears upon clicking. While using a combo button prevents adding an extra step in the process, it can add friction to the user decision making process when the options in the dropdown list are not intuitive enough. |
-| Isolated  | Used for extra actions which are, as the name suggests, isolated in an interface. Those button do not have borders and background colors, in order to reduce noise and to make them less conspicuous. Icons are optional and may help users identify actions faster. However, keep in mind that too many icons can be as distracting and confusing as too few.                                                                                                                                |
-| Icon Only | Used when there is a lack of space and a high density of content. This button type is preferably used to represent labels that are easily represented iconically, such as an icon "..." for "more items" (see [Menu button](../#/components/Menu) for an example).                                                                                                                                                                                                                            |
+
+## Variants
+
+| **Type** | **Purpose** |
+| **Primary** | Use primary buttons for the main action you want to draw the user attention to. There should be only one primary action in a section or page. |
+| **Secondary** | Use secondary buttons to display action of lesser importance or to display multiple actions of similar importance. |
+| **Left icon + label** | Add a left icon to add quick and visual clarity to what the action is or draw attention to the button. Make sure the icon is highly recognizable. |
+| **Left icon + label** | Use the right icon to indicate that additional options are available upon selecting the button. This means that the action will not be carried right away upon clicking the button, but a menu overlay would be displayed where the user can select the exact action to be performed. For instance, when the action is "Create" and options are to create from a template or a blank file. |
+| **Right icon + label** | Use only an icon when space is an issue or when the icon is highly and instantly recognizable. For instance, the "Setting" button is commonly presented only using a cog icon. Never use icon only button for primary actions. |
+| **Append and prepend separator** | Adding a separator is an aesthetic choice especially used when two or more buttons are stacked vertically. Use a separator to align the icon to the right or left or the button. Note that all buttons must have the same width when vertically stacked. |
```

Comment:
All this section is really hard to read, we're you attempting to make a table? :eyes: 
![image](https://user-images.githubusercontent.com/63734941/152883150-e309f688-c347-4129-91c8-43a23c8abf4a.png)

Reply:
Here's the syntax for a table, you might be just missing the line `| ---- | --- | ---- |` to delimit the table header

![image](https://user-images.githubusercontent.com/35579930/153032628-3e178f49-49b5-4829-b84c-ac04db59b863.png)

Reply:
I fix it. let me know if it's better

Reply:
fixed it for real (I think)